tests/out_test.go

```diff
@@ -118,5 +118,35 @@ var _ = Describe("Out", func() {
 
 			Expect(session.Err).To(gbytes.Say(docker("pull 123123.dkr.ecr.us-west-2.amazonaws.com:443/testing")))
 		})
+
+		Context("When using ECR with dry_run", func() {
```

Comment:
This behavior isn't related to ECR; I wouldn't put it under that context. Can it be tested without it?